src/main/kotlin/gg/essential/multi-version.gradle.kts

```diff
@@ -28,7 +28,15 @@ afterEvaluate { configureResources() } // delayed because it needs project.versi
 parent?.let(::inheritConfigurationFrom)
 
 fun setupLoomPlugin() {
-    extra.set("loom.platform", if (platform.isForge) "forge" else "fabric")
+    extra.set(
+        "loom.platform",
+        when {
```

Comment:
Could this just be `platform.loaderStr`?

Reply:
Yeah I overlooked that, my bad